railties/lib/rails/engine.rb

```diff
@@ -113,7 +112,7 @@ module Rails
   # == Endpoint
   #
   # An engine can also be a Rack application. It can be useful if you have a Rack application that
-  # you would like to wrap with +Engine+ and provide with some of the +Engine+'s features.
+  # you would like to wrap with +Engine+ and provide some of the +Engine+'s features.
```

Comment:
```suggestion
  # you would like to provide with some of +Engine+'s features.
```

Reply:
Awesome, updated to this, so much clearer!